.github/workflows/test.yml

```diff
@@ -17,19 +17,19 @@ jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         entry:
-          - { ruby: 2.4, allowed-failure: false }
-          - { ruby: 2.5, allowed-failure: false }
-          - { ruby: 2.6, allowed-failure: false }
           - { ruby: 2.7, allowed-failure: false }
-          - { ruby: 3.0, allowed-failure: false }
+          - { ruby: '3.0', allowed-failure: false }
```

Comment:
Let's quote all versions to avoid this problem?

Reply:
Sure